### ISSUE TITLE
Predicted probabilities SVM

### ISSUE THREAD

**Issue by DaphneGroot** [simondewit/pickleteam#13]
For the last couple of days I've been working on the ensemble method we discussed during the last meeting, and I think the overall method works. The only problem left is obtaining the predicted probabilities for the SVM method. 
On the website of sklearn it says that the function `predict_proba` only works with the loss function 'log', but we use 'hinge' (using 'log' lowers the score extremely).

Any ideas how the probability scores can be obtained?

**Comment by hslh** [comment]
Hmm, I hadn't thought of that! According to [https://stackoverflow.com/a/20778851](https://stackoverflow.com/a/20778851), it is possible by using the SVC implementation for SVMs in sklearn. I'm not sure if that gives you all the same options and performance, so you should test that. There are also some caveats about the probabilities produced by this method, as linked in that post. 

Alternatively, you could try to use the confidence scores from the SVM in the SGDClassifier implementation, which should be available.  But I'm not sure how informative those are exactly, and how to combine those with the LSTM probabilities.
